Makefile

```diff
@@ -74,6 +74,15 @@ endif
 
 KERNEL_NAME := $(shell uname -s | tr A-Z a-z)
 
+# When PKG_CACHE_PATH is defined this means we're running in CI. PKG_CACHE_PATH
```

Comment:
`PKG_CACHE_PATH` is for vercel/pkg, and kind of super unrelated to this, and an implementation detail of our CLI build.

Detecting CI can also be done with the BUILDKITE_ env vars, such as `BUILDKITE_COMMIT` :).

May be a little nicer, for next time when we change something here.